server/server.py

```diff
@@ -55,6 +77,22 @@ def submit():
     return resp
 
 
+@app.route(
+    "/run",
+    methods=["POST"],
+)
+def run():
+    test_location = os.getenv("TESTS_DIR")
```

Comment:
Same q: here about if this needs to be documented somewhere (i briefly checked root README in main, but didn't see a mention). Also, does this need a fallback value if the env var is undefined?

Reply:
I'll file something to update the docs